bridgetown-website/src/_components/documentation/multilang.rb

```diff
@@ -0,0 +1,20 @@
+class Documentation::Multilang < Bridgetown::Component
+  def template
+    split_content = content.split("===")
+
+    html -> { <<~HTML
+      <sl-tab-group>
+        <sl-tab slot="nav" panel="erb">ERB</sl-tab>
```

Comment:
What about outsourcing this code as a [Bridgetown plugin](https://www.bridgetownrb.com/plugins) with user-defined languages? This would be great for many projects :)

Reply:
I would love that! I think that can easily be done as a follow up a little down the road, or someone else's welcome to beat me to the punch. ☺️